FRBDK/Glue/Glue/CodeGeneration/CustomVariableCodeGenerator.cs

```diff
@@ -833,10 +831,10 @@ public static ICodeBlock AppendAssignmentForCustomVariableInElement(ICodeBlock c
             //if (!customVariable.SetByDerived && 
             else if (customVariable.DefaultValue != null && // if it's null the user doesn't want to change what is set in the file or in the source object
                 !customVariable.IsShared && // no need to handle statics here because they're always defined in class scope
-                !IsVariableTunnelingToDisabledObject(customVariable, glueElement)
+                !IsVariableTunnelingToDisabledObject(customVariable, saveObject)
```

Comment:
"saveObject" is a general term for any class that has gets serialized. We should be explicit when we can, so glueElement is better in this context.